src/main/java/uk/gov/ch/repository/capital/StatementOfCapitalRepository.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.ch.repository.capital;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.PreparedStatementSetter;
+import org.springframework.stereotype.Repository;
+import uk.gov.ch.model.capital.StatementOfCapital;
+
+import java.util.List;
+
+@Repository
+public class StatementOfCapitalRepository {
+
+    static final String STATEMENT_OF_CAPITAL_SQL = "SELECT coalesce(ssc.non_standard_share_class,sct.share_class_type_desc) as classOfShares, ctp.currency_type_code as currency,"
```

Comment:
Has this query been vetted by the DB team?

Reply:
Yes it has been approved by Mike Williams.